### DISCUSSION TITLE
v5 utilities API backport to v4

### DISCUSSION THREAD

**Discussion by pricop** [twbs/bootstrap#35627] [Ideas] [answered]
Hello guys,

Happy new year!

I was wondering if there are any plans to backport the v5 utilities API to v4. Seems like a rather straight forward backport. I was able to extract the API from v5 API and backport it to v4 on my machine (minus the rfs implementation) and works brilliant, but would love to see it by default in v4. Any plans for it?

**Comment by mdo** [comment, marked as answer]
Missed this question while I was out on medical leave and just randomly saw it in my notifications pile. No plans to do this :).

**Reply by pricop** [reply]
Would you be open to accept a PR if I was to do this  (including the Documentation part), or there's just no plans to have this for v4 at all? In either case, it's understandable. Take care!

**Reply by mdo** [reply]
No plans to add that kind of feature to v4. We stopped working on new features for v4 when v5 shipped in May 2021.
